libbeat/cmd/instance/metrics.go

```diff
@@ -2,19 +2,25 @@ package instance
 
 import (
 	"runtime"
+	"time"
 
 	"github.com/elastic/beats/libbeat/monitoring"
 )
 
 type memstatsVar struct{}
 
+type fixedstatsVar struct{}
+
 var (
 	metrics = monitoring.Default.NewRegistry("beat")
 )
 
 func init() {
 	var ms memstatsVar
 	metrics.Add("memstats", ms, monitoring.Reported)
+
+	var fs fixedstatsVar
+	metrics.Add("info", fs, monitoring.Reported)
 }
```

Comment:
one can use `monitoring.NewFunc` here:

```
func init() {
...
monitoring.NewFunc(metrics, "info", reportInfo, monitoring.Report)
...
}
```

with

```
func reportInfo(_ monitoring.Mode, V monitoring.Visitor) {
  V.OnRegistryStart()
  defer V.OnRegistryFinished()
  monitoring.ReportInt(V, "uptime", time.Since(startTime).Seconds())
}
```